concordium-sdk/src/main/java/com/concordium/sdk/transactions/AccountAddress.java

```diff
@@ -32,6 +38,25 @@ public static AccountAddress from(String address) {
         return AccountAddress.from(addressBytes);
     }
 
+    public AccountAddress newAlias(int alias) {
+        if (alias < 0) {
+            throw new NumberFormatException("Alias must be non negative");
+        }
+        if (alias > ALIAS_MAX_VALUE) {
+            alias = alias % ALIAS_MAX_VALUE;
```

Comment:
This is a bad idea. I think you should raise an exception since otherwise they will silently get duplicate aliases.

Reply:
I agree. I'll throw an exception to notify the user it has inputted a too large 'alias'.